docs/rules/spec.md

```diff
@@ -12,6 +12,15 @@ Ensures that your API document conforms to the [OpenAPI specification](https://s
 | 3.0 | ✅            |
 | 3.1 | ✅            |
 
+| ARAZZO | Compatibility |
```

Comment:
I added Arazzo and AsyncAPI section to start discussion on refactoring the structure.
This example with `spec.md` displays the biggest confusion as all supported standards have `spec` rule, but this page is reserved for OAS.

cc: @lornajane , @tatomyr

Reply:
```suggestion
| Arazzo | Compatibility |
```

Reply:
It is confusing, and quite a few of the rules work for more than one format (like info-contact I think for AsyncAPI)